MapleServer2/Types/SkillCast.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using MapleServer2.Data.Static;
+
+namespace MapleServer2.Types
+{
+    public class SkillCast
+    {
+        #region Declaration
+        private long SkillSN;
+        private int EntityId;
+        private int SkillId;
+        private short SkillLevel;
+        private int UnkValue; // unknown value
+        private double SkillDamage = 0;
+
+        public long GetSkillSN()
+        {
+            return SkillSN;
+        }
+
+        public int GetSkillId()
+        {
+            return SkillId;
+        }
+
+        public short GetSkillLevel()
+        {
+            return SkillLevel;
+        }
+
+        public int GetUnkValue()
+        {
+            return UnkValue;
+        }
```

Comment:
Also here, why not use `{ get; }` ?

Reply:
sure!, it was a way to understand what i was doing, i can simplify.